src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/scheduler/removesolicitorfromcase/RemoveSolicitorFromCaseTaskTest.java

```diff
@@ -0,0 +1,170 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.scheduler.removesolicitorfromcase;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.SearchResult;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.CcdDataStoreService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.CcdService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.SystemUserService;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType.AMEND_CASE_CRON;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType.CONSENTED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType.CONTESTED;
+
+@ExtendWith(MockitoExtension.class)
+class RemoveSolicitorFromCaseTaskTest {
+
+    @InjectMocks
+    private RemoveSolicitorFromCaseTask removeSolicitorFromCaseTask;
+
+    @Mock
+    private RemoveSolicitorFromCaseFileReader removeSolicitorFromCaseFileReader;
+    @Mock
+    private SystemUserService systemUserService;
+    @Mock
+    private CcdService ccdService;
+    @Mock
+    private CcdDataStoreService ccdDataStoreService;
+
+    @Test
+    void givenTaskNotEnabledWhenRunThenDoesNothing() {
+        removeSolicitorFromCaseTask.setEnabled(false);
+
+        removeSolicitorFromCaseTask.run();
+
+        verifyNoInteractions(removeSolicitorFromCaseFileReader);
+        verifyNoInteractions(systemUserService);
+        verifyNoInteractions(ccdService);
+        verifyNoInteractions(ccdDataStoreService);
+    }
```

Comment:
Could line 47 be removed? It would be nice to know that the https://github.com/value default of false is working as we expect.